cros_gralloc/cros_gralloc_driver.cc

```diff
@@ -19,6 +22,12 @@
 #include "i915_private_android.h"
 #endif
 
+#define IVSH_WIDTH 0x780  // 0x780-1920  0xA00-2560
+#define IVSH_HEIGHT 0x438 // 0x438-1080   0x5A0-1440
```

Comment:
it is better directly use decimal instead of hex.
Also we need think over if there is better solution for that.